src/swift/Sources/Models/TypeModel.swift

```diff
@@ -66,9 +65,21 @@ struct TypeModel {
         }
     }
 
+    init(from source: PatternInitializer) {
+        if case let identPattern as IdentifierPattern = source.pattern,
+            let typeAnnotation = identPattern.typeAnnotation {
+            self.init(from: typeAnnotation)
+        } else if let expression = source.initializerExpression as? LiteralExpression {
+            self.init(from: expression.kind.textDescription)
+        } else {
+            SharedLogger.fail("Unable to extract type information.")
+        }
+    }
+
```

Comment:
I'm not really sure what information is attempting to be extracted.

Reply:
You'd need to follow the other inits. However, line 73 is the initializer using just a plain string that you asked about above. The type info we are trying to extract is that shown in the `TypeModel` declaration.